haiku-games/benettris/benettris-0.2.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="Tetris client and server"
+DESCRIPTION="BeNetTris is a multiplayer Tetris game that you can play \
+with your friends! BeNetTris is client and server compatible with V1.13 \
+or V1.14 of Tetrinet or Tetrinet TetriFast."
+HOMEPAGE="https://github.com/HaikuArchives/BeNetTris"
+COPYRIGHT="2000-2008 Cedric Vincent"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="5d04121340b7c730160d5eab6a97c79b97168464"
+SOURCE_URI="https://github.com/HaikuArchives/BeNetTris/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="bd3096b84601a257cd22818ba0396be600d4b99e2f9d02f000d029502af9eddc"
+SOURCE_FILENAME="BeNetTris-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="BeNetTris-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	benettris = $portVersion
+	app:BeNetTris = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	cmd:find
+	"
+
+BUILD()
+{
+	find . -exec touch {} \;
```

Comment:
This should be moved to `PATCH()` instead.

Reply:
A clock skew is rather scare, have you checked if the clock on your machine was correct?

Reply:
> There's this problem with `Makefile having a modification time 19648s in the future` :(

What happened here is that your system is not using the right timezone and/or time, so when you grab a tarball which contains files recently modified, they have a timestamp which appears to be in the future.
I guess your timezone is UTC instead of UTC+x (with x being probably something between 9 and 11).

BTW, once you have fixed your clock settings you will probably see that you don't need to touch the files.

Reply:
@bach5000: Okay. I found what your real timezone is. It is correctly set in one (or some) of your systems, but not in all. One (or maybe more) of your systems is using UTC as its timezone. That's Okay to use this, but if you do that then you have to set your system's clock accordingly, i.e. to the real UTC time, not the localtime.